widget/accordion.go

```diff
@@ -121,10 +121,29 @@ func (r *accordionRenderer) Layout(size fyne.Size) {
 	dividerOff := (theme.Padding() + theme.SeparatorThicknessSize()) / 2
 	x := float32(0)
 	y := float32(0)
+	hasOpen := 0
+	for i, ai := range r.container.Items {
+		h := r.headers[i]
+		min := h.MinSize().Height
+		y += min
+
+		if ai.Open {
+			y += theme.Padding()
```

Comment:
You might want to cache the padding here (above and below as well) to avoid the theme lookup each time, like we have done in many other places.

Reply:
Thanks, fixed (and in the older MinSize method too)